README.md

```diff
@@ -155,8 +155,8 @@ try project.write(path: "MyApp.xcodeproj")
 #### Adding `Home` group inside `Sources` group
 
 ```swift
-guard var sourcesGroup = project.pbxproj.objects.groups.first(where: {$0.value.name == "Sources"})?.value else { return }    
-let homeGroup = PBXGroup(reference: "xxx", children: [], sourceTree: .group, path: "Home")
+guard var sourcesGroup = project.pbxproj.objects.groups.first(where: {$0.value.name == "Sources" || $0.value.path == "Sources"})?.value else { return }    
+let homeGroup = PBXGroup(reference: project.pbxproj.generateUUID(for: PBXGroup.self), children: [], sourceTree: .group, path: "Home")
```

Comment:
I've spent some time searching for a way to generate references until found this method.

Reply:
Yeah we introduced it in the version 2.0 but we forgot to update the readme. The function is `generateReference` instead of `generateUUID` and it should be used like this:

```
project.pbxproj.generateReference(PBXGroup.self, "home")
```

The second argument must be a string that uniquely identifies the object. In this case we can just use "home".

Reply:
Yeah, looks like I was using 1.7.0 version

Reply:
Unfortunately it means that readme examples are completely out of date. I would suggest to keep section with examples for 1.x version and with 2.x

Reply:
Thank you very much @ilyapuchka